### PR TITLE
Fix missing space in ru translation

### PR DESCRIPTION
**Pull request by ACK1D**

### CODE REVIEW

**Comment by ACK1D** [comment]
@zerolab can you help with ruff checks?

**Comment by zerolab** [comment]
@ACK1D yeah. it is on my list to update https://github.com/torchbox/wagtailmedia/blob/main/.github/workflows/ruff.yml#L18 to install the same version defined in https://github.com/torchbox/wagtailmedia/blob/main/.pre-commit-config.yaml#L21 and add comments to both to keep them in sync

**Comment by ACK1D** [comment]
@zerolab checked on version 0.2.0, no problems. But the checks do not pass. That's weird.

**Comment by zerolab** [comment]
the ruff check currently installs the latest ruff version, hence the failure.

the test failures are separate, though

**Comment by ACK1D** [comment]
commit [718151a](https://github.com/torchbox/wagtailmedia/pull/245/commits/718151ab9b5637f9dae33629541d3349718ed0b4) includes fixes for which ruff 0.5.4 was "swearing". I don't know if this is necessary. I just wanted to correct a line in the translation)

File: tests/urls.py [hunk 1]
```diff
@@ -18,7 +18,7 @@
     path("", include(wagtail_urls)),
 ] + [
     path(
-        "%s<path:path>" % prefix.lstrip("/"),
+        f"{prefix.lstrip("/")}<path:path>",
```
Comment (zerolab) [inline tests/urls.py:21]:
```suggestion
        f"{prefix.lstrip('/')}<path:path>",
```

**Comment by ACK1D** [comment]
@zerolab well done :)

**Pull request merged**